webrender_traits/src/api.rs

```diff
@@ -18,6 +18,12 @@ use {WebGLCommand, WebGLContextId};
 pub type TileSize = u16;
 
 #[derive(Clone, Deserialize, Serialize)]
+pub enum ScrollClamping {
```

Comment:
let's move it after `ApiMsg` definition, to match what over used enums are doing